sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/DigitalTwinsRelationshipAsyncTest.java

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we just call it RelationshipsAsyncTest? do we need the DigitalTwin prefix?

Reply:
I followed the naming pattern that we've used in our .NET samples